docs/developers/Custom_Rules.mdx

```diff
@@ -331,6 +330,7 @@ export const rule = createRule({
   meta: {
     docs: {
       description: 'Avoid looping over enums.',
+      requiresTypeChecking: true,
```

Comment:
Maybe add a comment saying what it's useful for

Reply:
I can't think of anything to say that isn't implied by the name. 😕

Reply:
I meant for example: the rule tester and the doc generator use this info to do xyz, etc.

Reply:
Oh right now that we have the custom `meta.docs` types, this isn't even needed anymore! I'll just remove it.